test/cmdlineTests/avoid_double_cleanup/err

```diff
@@ -0,0 +1,17 @@
+Warning: SPDX license identifier not provided in source file. Before publishing, consider adding a comment containing "SPDX-License-Identifier: <SPDX-License>" to each source file. Use "SPDX-License-Identifier: UNLICENSED" for non-open-source code. Please see https://spdx.org for more information.
```

Comment:
Can you try to fix this so it does not contain warnings?

Reply:
Although I'm not sure if this is really the right kind of test for this issue.

Reply:
Do you have a suggestion for a better test?

Reply:
The test should be much more focused. Maybe you could count the number of `and` opcodes or check the code generated for a specific expression only (we have other tests of both kinds).